pkg/kubectl/cmd/convert.go

```diff
@@ -62,7 +62,7 @@ var (
 // NewCmdConvert creates a command object for the generic "convert" action, which
 // translates the config file into a given version.
 func NewCmdConvert(f cmdutil.Factory, out io.Writer) *cobra.Command {
-	options := &ConvertOptions{}
+	options := &ConvertOptions{local: true}
```

Comment:
Oh, this is neat.  Can you create a `NewDefaultConvertOptions() *ConvertOptions` function that does this?